cpp/models/abm/README.md

```diff
@@ -1,57 +1,127 @@
 # Agent Based Model
 
-This module models and simulates the epidemic using an agent based model (*ABM*) approach. Unlike the SECIR compartment model that uses an ODE, this model simulates the spread of COVID19 in a population with actual persons (the agents) moving throughout the world and interacting with each other.
+This module models and simulates the epidemic using an agent based model (*ABM*) approach. Unlike the SECIR compartment model that uses an ODE, this model simulates the spread of COVID19 in a population with actual persons (the agents) moving throughout locations in the world and interacting (infecting) with each other.
```

Comment:
```suggestion
This module models and simulates the epidemic using an agent based model (*ABM*) approach. Unlike the [SECIR](../ode_secir/README.md) compartmental model that uses a system of ODEs, this model simulates the spread of COVID19 in a population with discrete persons (the agents) moving throughout locations in the world and interacting with (infecting) each other.
```
A few suggestions:
- Link to the secir readme. Could also extend the link to 'SECIR comartmental model'
- I would call it is a system of ODEs, not just one, but I think technically either is correct
- Since we do not do human experiments, 'actual persons' is in my opinion the wrong term to use here. 'discrete' is (while correct) maybe not the best descriptor either, we could discuss this.
- This last one may be personal preference, but I read it as '[...] infecting with each other', which is weird. 

Since these changes are in a single line, I added them in one suggestion, but please do consider them separately.